dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -2126,7 +2126,4 @@ div#tryHoc {
     color: #fff;
     text-decoration: none;
   }
-  img#hocLogo {
```

Comment:
actually it's best to have a higher resolution image, especially for retina users.

on pegasus we'd store a really high-res image as a nice master source, and then use its fit-40 (or fit-80 to be retina friendly) URL-scheme to retrieve a server-resized image that's appropriate for the usage.

i don't think dashboard has such a facility but you could reach across to pegasus for that.

but at any rate, it'd be good to be a bit higher resolution again.